slices/libunwind-13.yaml

```diff
@@ -4,5 +4,7 @@ slices:
     essential:
       - libc6_libs
     contents:
-      /usr/lib/llvm-13/lib/libunwind.so.*:
-      /usr/lib/x86_64-linux-gnu/libunwind.so.*:
+      /usr/lib/llvm-13/lib/libunwind.so.1.*:
+      /usr/lib/llvm-13/lib/libunwind.so.1:
```

Comment:
So libunwind8 ships .so.8 while libunwind-12 and -13 ship .so.1?  Gotta love these conventions.

Reply:
Yep \o/

Reply:
:man_facepalming: